src/video/khronos/KHR/khrplatform.h

```diff
@@ -99,15 +99,15 @@
  *-------------------------------------------------------------------------
  * This precedes the return type of the function in the function prototype.
  */
-#ifdef KHRONOS_STATIC
+#if defined(KHRONOS_STATIC)
```

Comment:
please keep this as is. it is the way khronos mainstream does it

Reply:
Similarly, this is going back to the content this file had in 72fe6cc8 "Updated to the latest version of OpenGL and Vulkan headers from the Khronos registry".

Reply:
I've confirmed that after this PR, `src/video/khronos/KHR/` is now identical to the `/usr/include/KHR/` on my Debian system.